src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/ITagHelper.cs

```diff
@@ -25,7 +25,7 @@ public interface ITagHelper
         /// </summary>
         /// <param name="context">Contains information associated with the current HTML tag.</param>
         /// <remarks>When more than one <see cref="ITagHelper"/> runs on the same element,
-        /// <see cref="TagHelperOutput.GetChildContentAsync"/> may be invoked prior to <see cref="ProcessAsync"/>.
+        /// <see cref="TagHelperOutput.GetChildContentAsync()"/> may be invoked prior to <see cref="ProcessAsync"/>.
```

Comment:
Actually :-1:  Something is wrong here. Should remain able to reference the various overloads. It's the method group that's intended here, not a specific method.